src/AdScreenerTool.js

```diff
@@ -240,13 +245,52 @@ const AdScreener = (params) => {
   	setSelectedTopicOrFullTextSearchTab(k);
   	if(k=='topics') {
   		setFullTextSearchQuery(null);
+		setPageId(null);
   	}
+	else if(k=='advertiser') {
+		setFullTextSearchQuery(null);
+		setTopic({ selectedOption: "" });
+		setTopicParam(undefined);
+	}
   	else {
-		setTopic({ selectedOption: ""});
+		setTopic({ selectedOption: "" });
 		setTopicParam(undefined);
+		setPageId(null);
   	}
   }
 
+  const [isLoading, setIsLoading] = useState(false);
+  const [options, setOptions] = useState([]);
+  const [pageId, setPageId] = useState(null);
```

Comment:
please move these declarations to the top of the component along with the other useState declarations.